src/pages/SurveyList.js

```diff
@@ -74,11 +78,9 @@ const SurveyList = () => {
           alignItems: 'center'
         }}>
         <Box sx={{ marginLeft: '12px' }}>{survey.id}.</Box>
-        <Box sx={{ flexGrow: 1 }}>
-          <Box>{survey.name}</Box>
-          <Typography>Number of questions: TODO?</Typography>
-          <Typography>Number of responses: TODO?</Typography>
-        </Box>
+        <Button sx={{ flexGrow: 1 }} onClick={()=> handleAnswerSurvey(survey.id)}>
+          {survey.name}
```

Comment:
This behavior is not obvious at all and doesn't really make sense from an end-user perspective

Reply:
Not sure what the issue is here? When I go to answer surveys it looks fine to me

Reply:
Same here, looks fine